cores/esp32/esp32-hal-uart.h

```diff
@@ -59,6 +59,8 @@ uint32_t uartAvailableForWrite(uart_t* uart);
 uint8_t uartRead(uart_t* uart);
 uint8_t uartPeek(uart_t* uart);
 
+void uartRxFifoToQueue(uart_t* uart);
```

Comment:
I would not export the this function.  It should only be call from inside `esp32-hal-uart.c`

Reply:
Agree, I just wasn't sure whether all .c functions needed to be in .h because of unexpected usage.
I'll remove from .h